features/trad_ssh_push.feature

```diff
@@ -68,7 +68,7 @@ Feature: Register a traditional system to be managed via SSH push
     Given I am on the Systems page
     And I run "sed -i '/127.0.1.1/d' /etc/hosts" on "sle-client"
     And I run "rm /srv/www/htdocs/pub/bootstrap/bootstrap-ssh-push-tunnel.sh" on "server"
-    And I remove server hostname from hosts trad-client
+    And I remove server hostname from hosts file on "sle-client"
```

Comment:
the whole scenario can be removed. speak wit @mateiw  about this ( he introduced the cleanup, so we don't need this scenario for removing the host